charts/mgmt-agent/values.yaml

```diff
@@ -70,7 +70,7 @@ deployment:
       # Files created in the Container will use group ID 2000, replace it with a different value if desired
       fsGroup: 2000
 
-   # By default Agent will be deployed as StatefulSet. For DaemonSet deployment, set this to true
+   # Set the daemonset deployment to true, to deploy additional agents, to distribute the node metrics collection
```

Comment:
Setting the daemonset deployment to true, will deploy the Management Agents as a daemonset in addition to deploying the Management Agent as a statefulset. This is done to to distribute the node metrics collection to agents running on the node

Reply:
done